### DISCUSSION TITLE
Override prefers-color-scheme

### DISCUSSION THREAD

**Discussion by Flashbin** [Eloston/ungoogled-chromium#1834] [Q&A] [answered]
Is there a way to permanently override _prefers-color-scheme_? Would be helpful since at least on my system (Linux Mint 20.3), chromium doesn't seem to honor my dark mode theme and just displays all web sites in light mode. And some sites don't have an option to switch.

**Comment by networkException** [comment, marked as answer]
Some of the flags listed here should set that. https://github.com/networkException/dotfiles/blob/7149668094e1143e31ffb6bf40c3d4386b22bbda/desktop/chromium.desktop#L9

**Reply by Flashbin** [reply]
Thanks alot!

For anyone doing this with the Flatpak version:
Add
```
--enable-features=WebUIDarkMode
--force-dark-mode
```
to `~/.var/app/com.github.Eloston.UngoogledChromium/config/chromium-flags.conf`

**Reply by networkException** [reply]
ah you were talking about flatpak, yea thats also documented in https://github.com/flathub/com.github.Eloston.UngoogledChromium#how-to-force-enable-dark-theme

**Reply by PF4Public** [reply]
Just a FYI for anyone coming here.
AFAIK `prefers-color-scheme` is broken in Chromium on Linux (on GTK at least) with no interest from Google to fix it (unlike Windows). I apply this patch to make it work on Gentoo: https://github.com/PF4Public/gentoo-overlay/blob/master/www-client/ungoogled-chromium/files/gtk-fix-prefers-color-scheme-query.diff
